study.md

```diff
@@ -27,17 +27,23 @@ it's also not a rendered page.
 Please name two things the DOM is not.
 
 ```md
-<!-- your answer here -->
+<!-- The DOM is not a source or a rendered page.
+
+source : the README file in this repo. -->
 ```
```

Comment:
When they say DOM is not the source, it's referring to how it's not our source code or the HTML we initially write. What we see when we click 'View Source' on a webpage is not the DOM.